services/bower/bower-license.service.js

```diff
@@ -45,11 +44,7 @@ module.exports = class BowerLicense extends LegacyService {
           json: true,
           uri: `https://libraries.io/api/bower/${repo}`,
         }
-        if (serverSecrets && serverSecrets.libraries_io_api_key) {
-          options.qs = {
-            api_key: serverSecrets.libraries_io_api_key,
-          }
-        }
```

Comment:
This was added in #919 to avoid rate limits in production. What's the reasoning for removing it?

Reply:
This is explained in the commit message for 31c253f :

> The libraries.io docs claim you need to be authenticated to make any API request: https://libraries.io/api#authentication

> In practice we can call https://libraries.io/api/bower/jquery just fine with no token and based on chucking a load of requests at it and examining the `x-ratelimit-remaining` headers you actually seem to get a better limit with no authentication.

> All of our libraries.io badges in `services/librariesio` seem to have been running fine with no token for some time.

I'm not exactly sure how quickly its resetting the limit, but with no auth you seem to get 30 new requests every second (or possibly some interval less than a second) whereas with a token you're restricted to only 60 requests per minute (the documented limit). I know the docs say you must authenticate, but you can check it with `curl -I`.